system-addon/content-src/lib/snippets.js

```diff
@@ -373,6 +374,8 @@ export function addSnippetsSubscriber(store) {
     // state.Snippets.initialized             Is the snippets data initialized?
     // snippets.initialized:                  Is SnippetsProvider currently initialised?
     if (state.Prefs.values["feeds.snippets"] &&
+      // If the message center experiment is enabled, don't show snippets
+      !state.Prefs.values.messageCenterExperimentEnabled &&
```

Comment:
Somehow I still get them.
<img width="1290" alt="image" src="https://user-images.githubusercontent.com/810040/37911858-72bc964a-3111-11e8-9dea-2dfd2b100d85.png">
<img width="728" alt="image" src="https://user-images.githubusercontent.com/810040/37911887-8027df74-3111-11e8-8591-770a6c9b7546.png">

Reply:
I guess we have to call uninit on the snippets when we init the message center?

Reply:
Ah, those are onboarding – I need to find a way to turn that off too

Reply:
I added onboarding overriding to the feed, so this should be handled appropriately now.